src/background.js

```diff
@@ -10,3 +10,27 @@ browser.runtime.onInstalled.addListener(function (details) {
     })
   }
 })
+
+function onCreated() {
+  if (browser.runtime.lastError) {
+    console.log(`Error: ${browser.runtime.lastError}`);
+  } else {
+    console.log("Item created successfully");
+  }
+}
+
+browser.contextMenus.create({
+  id: 'scite-citation-search',
+  title: 'Search scite.ai citation statements',
+  contexts: ['selection']
+}, onCreated)
+
+browser.contextMenus.onClicked.addListener(function (info, tab) {
+  if (info.menuItemId === 'scite-citation-search') {
+    if (info.selectionText) {
+      browser.tabs.create({
+        url: `https://scite.ai/search/citations?q=${info.selectionText}&utm_source=generic&utm_medium=plugin&utm_campaign=plugin-citation-search`
```

Comment:
@bananaoomarang I think you will want to URI encode this text otherwise a user could select illegal text I think

Reply:
weiird chrome automatically encodes it and firefox doesnt

Reply:
Just updated for this I think